.github/workflows/tests.yml

```diff
@@ -76,3 +76,25 @@ jobs:
         # XDG_RUNTIME_DIR is set.
         # TODO: figure out what exactly podman needs
         sudo -E XDG_RUNTIME_DIR= pytest-3 -s -vv
+
+  integration-macos:
+    name: "Integration macos"
+    # needed to get latest cpu
+    runs-on: macos-13
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Setup up python
+      uses: actions/setup-python@v4
+    - name: Setup up podman
+      run: |
+        brew install podman netcat
+        podman machine init --rootful
+        podman machine start
+    - name: Install test dependencies
+      run: |
+        sudo pip install pytest flake8 paramiko
```

Comment:
Would you mind using a requirements.txt file to list the packages with their corresponding pinned versions?

Reply:
Thanks, good point. Let me fix this.